packages/mui-base/src/useAutocomplete/useAutocomplete.js

```diff
@@ -1007,13 +1007,25 @@ export default function useAutocomplete(props) {
 
   // Prevent input blur when interacting with the combobox
   const handleMouseDown = (event) => {
+    const isToolTipElement = event.target?.parentElement?.getAttribute('role')?.includes('tooltip');
```

Comment:
I think that this check is too specific.

Reply:
Yes, see totally your point.
I'm working on it and looking at that other issues.

Reply:
But regards to this issue only, I made a change.